qemu/tests/expose_host_mtu.py

```diff
@@ -95,24 +94,48 @@ def cleanup_ovs_ports(netdst, ports):
     session = vm.wait_for_login(timeout=login_timeout)
 
     host_ip = utils_net.get_ip_address_by_interface(params["netdst"])
-    if os_type == "linux":  # TODO, will support windows later
+    if os_type == "linux":
         session.cmd_output_safe(params["nm_stop_cmd"])
         guest_ifname = utils_net.get_linux_ifname(session,
                                                   vm.get_mac_address())
         output = session.cmd_output_safe(
-                params["check_guest_mtu_cmd"] % guest_ifname)
+            params["check_guest_mtu_cmd"] % guest_ifname)
         error_context.context(output, logging.info)
         match_string = "mtu %s" % params["mtu_value"]
-        if match_string in output:
-            logging.info("Host mtu %s exposed to guest as expected!" %
-                         params["mtu_value"])
-            logging.info("Ping from guest to host with packet size 3972")
-            status, output = utils_test.ping(host_ip, 10, packetsize=3972,
-                                             timeout=30, session=session)
-            ratio = utils_test.get_loss_ratio(output)
-            if ratio != 0:
-                test.fail("Loss ratio is %s", ratio)
-        else:
+        if match_string not in output:
             test.fail("host mtu %s not exposed to guest" % params["mtu_value"])
+    elif os_type == "windows":
+        connection_id = utils_net.get_windows_nic_attribute(
+            session, "macaddress", vm.get_mac_address(), "netconnectionid")
+        output = session.cmd_output_safe(
+            params["check_guest_mtu_cmd"] % connection_id)
+        error_context.context(output, logging.info)
+        lines = output.strip().splitlines()
```

Comment:
May be this will be helpful 
https://stackoverflow.com/questions/10711116/strip-spaces-tabs-newlines-python

Reply:
Thanks. But the input string is like:
   MTU  MediaSenseState   Bytes In  Bytes Out  Interface
------  ---------------  ---------  ---------  -------------
  4000                1     744188     181786  Ethernet

I need to get the index of MTU, which is 1, to get the value of index 1, which is 4000.
Since there're one space in some of the items, like "Bytes In" and "Bytes Out", I cannot remove space at the beginning but to collect how many items are there by using "  "(two space) as separation.
Anyway, thanks for your info.